tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -82,13 +87,31 @@ jobs:
         --test_apk_path "androidTest/debug/app-debug-androidTest.apk" \
         --devices "Samsung Galaxy S23-13.0" "Google Pixel 3-9.0"
     displayName: Run E2E tests using Browserstack
+    condition: eq('${{ parameters.runBrowserStackTests }}', 'true')
     workingDirectory: $(Build.BinariesDirectory)/android_test/android/app/build/outputs/apk
     timeoutInMinutes: 15
-    enabled: false
     env:
       BROWSERSTACK_ID: $(browserstack_username)
       BROWSERSTACK_TOKEN: $(browserstack_access_key)
 
+  - ${{ if eq(parameters.runBrowserStackTests, false) }}:
+    - template: install-appcenter.yml
+
+  - script: |
```

Comment:
Should you add one level indentation here? If appcenter was not installed, it should not run appcenter tests.

Reply:
oh true -- the script step has a condition that checks the runBrowserStackTests parameter, but will be more readable and achieves the same goal if script step is indented